test/utils/consts.go

```diff
@@ -13,14 +12,17 @@ const (
 	PtpLinuxDaemonNamespace         = "openshift-ptp"
 	PtpOperatorDeploymentName       = "ptp-operator"
 	PtpDaemonsetName                = "linuxptp-daemon"
-	PtpSlaveNodeLabel               = "ptp/test-slave"
 	PtpGrandmasterNodeLabel         = "ptp/test-grandmaster"
+	PtpSlaveNodeLabel               = "ptp/test-slave"
+	PtpBCMasterNodeLabel            = "ptp/test-bc-master"
+	PtpBCSlaveNodeLabel             = "ptp/test-bc-slave"
 	PtpResourcesGroupVersionPrefix  = "ptp.openshift.io/v"
 	PtpResourcesNameOperatorConfigs = "ptpoperatorconfigs"
 	NodePtpDeviceAPIPath            = "/apis/ptp.openshift.io/v1/namespaces/openshift-ptp/nodeptpdevices/"
 	ConfigPtpAPIPath                = "/apis/ptp.openshift.io/v1/namespaces/openshift-ptp/ptpconfigs"
 	PtpContainerName                = "linuxptp-daemon-container"
 	EventProxyContainerName         = "cloud-event-proxy"
+	NamespaceTesting                = "ptp-testing"
```

Comment:
why do we need ptp-testing ? everything should be within openshift-ptp right ?

Reply:
fixed